docs/ABOUT.md

```diff
@@ -0,0 +1,9 @@
+Nim is a general-purpose language that is designed to be efficient, expressive, and elegant. Nim is statically typed and compiled but lets you write elegant code that runs efficiently. Nim lets you target many platforms by compiling code to C, C++, JavaScript, or Objective-C. Nim syntax is similar to Python and shares many of Python's characteristics but draws inspiration from a number of languages, such as C, C++, C#, Lisp, Ada, Go, Oberon, and others.
+
+As Nim is a general-purpose language it can be used in various places but has been taking a real shine in the area of Game development and Systems Programming including Embedded systems.
+
+Nim gives you a lot of the convenience you find in Python and other languages, like easy to read syntax, a package management system for using new libraries in your projects, and user-friendly trackbacks for debugging errors.
+
+Nim gives you the speed and portability of C, but with a high-performance garbage-collecter and elegant syntax that makes the language very approachable to new people. Nim also gives you control, letting you change the garbage-collector or turn it off completely giving you the option to manage memory yourself. You have a variety of programming paradigms and abstraction layers with Nim.
+
+Designed and developed by Andreas Rumpf and first appearing in 2008, Nim is still considered to be a relatively niche programing language.
```

Comment:
s/niche/new/ :)

Reply:
@dom96  Well, I had new here before and it was pointed out that a decade isn't exactly new, so I switched it to niche. Is there some way to better word this?

Reply:
I think "relatively" takes care of that :)

What is "new" is very subjective. Maybe it would be better to just remove this last paragraph, it doesn't really add much and just puts Nim down.